src/controllers/articleController.js

```diff
@@ -35,7 +37,7 @@ class ArticleController {
       payload.authorId = id;
       const article = await Article.create(payload);
       const {
-        slug, category, images, tagList, authorId
+        slug, category, images, tagList, authorId, likes, dislikes
```

Comment:
I do not expect the payload when creating an article to have likes and dislikes, why are we getting them here?

Reply:
I wanted to give an article default likes and dislikes after being created. But, It's not necessary, I will display them only for the retrieval.